### PR TITLE
Fixed err msg handling, resolves #238

### LINKED ISSUE #238: NOTES - Intake Docs Rec'd Script - extra msgbox
There is a blank msgbox that is popping up after you click OK and run the script.

### PR DESCRIPTION
**Pull request by khiestand**
Blip - added err msg handling, resolves issue #238

### CODE REVIEW

File: Script Files/NOTES/NOTES - INTAKE DOCS RECEIVED.vbs [hunk 1]
```diff
@@ -92,15 +95,17 @@ If row <> 0 then
...
-DO
-	dialog intake_docs_recd_dialog
-	IF buttonpressed = 0 THEN stopscript
-	CALL PRISM_case_number_validation(PRISM_case_number, case_number_valid)
+Do
+	err_msg = ""
+	'Shows dialog, validates that PRISM is up and not timed out, with transmit
+	Dialog intake_docs_recd_dialog
+	If buttonpressed = 0 then stopscript
 	IF case_number_valid = False THEN error_msg = "Your case number is not valid. Please make sure it is in the following format: XXXXXXXXXX-XX.  "
```
Comment (RobertFewins-Kalb) [inline Script Files/NOTES/NOTES - INTAKE DOCS RECEIVED.vbs]: @khiestand, can you please update the variable name here as well? If it's not a valid case number, the user will not be informed and the script will continue with the erroneous case number.

File: Script Files/NOTES/NOTES - INTAKE DOCS RECEIVED.vbs [hunk 2]
```diff
@@ -92,15 +95,17 @@ If row <> 0 then
 	If isnumeric(left(PRISM_case_number, 10)) = False or isnumeric(right(PRISM_case_number, 2)) = False then PRISM_case_number = ""
 End if
 
-'Makes sure worker is in a valid PRISM Case
-DO
-	dialog intake_docs_recd_dialog
-	IF buttonpressed = 0 THEN stopscript
-	CALL PRISM_case_number_validation(PRISM_case_number, case_number_valid)
```
Comment (RobertFewins-Kalb) [inline Script Files/NOTES/NOTES - INTAKE DOCS RECEIVED.vbs]: @khiestand, are you sure you can get rid of the function call right here? We need a way of assigning a value to case_number_valid.

File: Script Files/NOTES/NOTES - INTAKE DOCS RECEIVED.vbs (hunk 1 above)
Comment (RobertFewins-Kalb) [inline Script Files/NOTES/NOTES - INTAKE DOCS RECEIVED.vbs]: @khiestand, can you please update the variable name here as well? If it's not a valid case number, the user will not be informed and the script will continue with the erroneous case number.

**Comment by khiestand** [comment]
@RobertFewins-Kalb I am going to hopefully have time on Monday at CS hackathon to finish this, might need your help!

**Comment by RobertFewins-Kalb** [comment]
I believe in you!

File: Script Files/NOTES/NOTES - INTAKE DOCS RECEIVED.vbs (hunk 2 above)
Comment (khiestand) [inline Script Files/NOTES/NOTES - INTAKE DOCS RECEIVED.vbs]: No I want to keep that in

**Comment by RobertFewins-Kalb** [comment]
I'm going to accept this and move it over to beta! Nice work!!

**Pull request merged**